Documentation/sources/content-card-ui/tutorial/displaying-content-cards.md

```diff
@@ -0,0 +1,139 @@
+
+### Display Content Cards in Compose UI application
+
+Below is an example of how to display content cards in a Compose UI application:
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+  super.onCreate(savedInstanceState)
+
+  binding = ActivityScrollingBinding.inflate(layoutInflater)
+  setContentView(binding.root)
+
+  val surfaces = mutableListOf<Surface>()
+  val surface = Surface("homepage")
+  surfaces.add(surface)
+
+  // Initialize the ContentCardUIProvider
+  contentCardUIProvider = ContentCardUIProvider(surface)
+
+  // Initialize the ViewModel
+  contentCardViewModel =
+  ViewModelProvider(this, AepContentCardViewModelFactory(contentCardUIProvider)).get(
+    AepContentCardViewModel::class.java
+  )
+
+  contentCardCallback = ContentCardCallback()
+
+  // Set a click listener for refresh button which calls the API for fetch content cards from Edge
+  val refreshButton: ImageButton = findViewById(R.id.refreshButton)
+  refreshButton.setOnClickListener {
+    Messaging.updatePropositionsForSurfaces(surfaces)
+    contentCardViewModel.refreshContent()
+  }
+
+  binding.composeView.apply {
+    setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+    setContent {
+      AppTheme {
+        AepContentCardList(contentCardViewModel)
+      }
+    }
+  }
+}
+
+@Composable
+private fun AepContentCardList(viewModel: AepContentCardViewModel) {
+  // Collect the state from ViewModel
+  val aepUiList by viewModel.aepUIList.collectAsStateWithLifecycle()
+  
+  // Create row with composables from AepUI instances
+  LazyRow {
+    items(aepUiList) { aepUI ->
+                               when (aepUI) {
+                                 is SmallImageUI -> {
+                                   val state = aepUI.getState()
+                                   if (!state.dismissed) {
+                                     SmallImageCard(
+                                       ui = aepUI,
+                                       style = smallImageCardStyleRow,
+                                       observer = ContentCardEventObserver(contentCardCallback)
+                                     )
+                                   }
+                                 }
+                               }
+                              }
+  }
+}    
+```
+
+Refer to this [TestApp](../../../../code/testapp/) for a complete example of how to display, customize and listen to UI events from content cards in a Compose UI application.
+
+#### Optional
+
+You may choose to order your Cards by the priority value entered in the AJO UI. To do this, sort the returned array:
+
+```kotlin
+// Reorder the AepUI list based on the ContentCardSchemaData fields if needed
+val reorderedAepUIList = aepUiList.sortedWith(compareByDescending {
+  val rank =
+  contentCardSchemaDataList[aepUiList.indexOf(it)]?.meta?.get("priority") as String?
+  ?: "0"
+  rank.toInt()
+})
+```
```

Comment:
nit: Since we're automatically doing this now, I think its fine to leave it out

Reply:
Can we instead add a section on getting the `ContentCardSchemaData` from `AepUI` using `ContentCardMapper`?